app/src/main/java/org/mozilla/fenix/AppRequestInterceptor.kt

```diff
@@ -10,17 +10,15 @@ import mozilla.components.browser.errorpages.ErrorType
 import mozilla.components.concept.engine.EngineSession
 import mozilla.components.concept.engine.request.RequestInterceptor
 import org.mozilla.fenix.components.metrics.Event
-import org.mozilla.fenix.exceptions.ExceptionDomains
 import org.mozilla.fenix.ext.components
 import org.mozilla.fenix.ext.settings
-import org.mozilla.fenix.ext.tryGetHostFromUrl
 
 class AppRequestInterceptor(private val context: Context) : RequestInterceptor {
-    override fun onLoadRequest(session: EngineSession, uri: String): RequestInterceptor.InterceptionResponse? {
-        val host = uri.tryGetHostFromUrl()
-
-        adjustTrackingProtection(host, context, session)
-
+    override fun onLoadRequest(
+        session: EngineSession,
+        uri: String
+    ): RequestInterceptor.InterceptionResponse? {
+        adjustTrackingProtection(context, session)
```

Comment:
I'm looking for a way to get rid of this. At the moment we need it. I'm doing some local tests, I will let you know if I find an alternative.

Reply:
I think this code should live on the ac, I will file an issue for finding an alternative.

Reply:
This is the ac issue https://github.com/mozilla-mobile/android-components/issues/4935
We are going to work on this issue next sprint.